app/views/episode_transcripts/transcripts/_new.html.erb

```diff
@@ -0,0 +1,91 @@
+<div class="col-12 mb-4" data-controller="upload" data-upload-disable-outlet="form">
+  <%# step 0: destroying transcript (will be nulled out if we actually replace it) %>
+  <%= form.hidden_field :id, data: {upload_target: "replace"} %>
+  <%= form.hidden_field :_destroy, value: true, data: {upload_target: "replace"} %>
+
+  <%# step 1: file field %>
+  <div class="input-group">
+    <div class="form-floating <%= "d-none" if upload_uploaded?(transcript) %>" data-upload-target="picker">
+      <% cls = "form-control position-absolute opacity-0 z-1" %>
+      <% data = {action: "change->unsaved#change upload#upload", upload_target: "field"} %>
+      <% data[:unsaved_target] = "changed" if transcript.marked_for_destruction? %>
+
+      <%# NOTE: name is nil, so we don't submit the actual file to Rails %>
+      <%= form.file_field :transcript_file, class: cls, accept: ".html,.json,.srt,.vtt", name: nil, data: data %>
```

Comment:
is this the correct assumption? that these are the only file types we should expect based on the podcast index examples? or should i have left it as the broader `text/plain` and `application/json`?

Reply:
Looks like `accept` works with both mime types and file extensions.  I think this is a good assumption to start - we'll see if anyone complains!

(Browsers also allow you to override this, and select any file you want.  So there is a workaround.)